### PR TITLE
Planning&Scripts: add ntp pool options and enrich planning information

### PR DESCRIPTION
**Pull request by Capri2014**

### CODE REVIEW

File: scripts/time_sync.sh [hunk 1]
```diff
@@ -20,10 +20,23 @@ sudo apt-get install -y ntpdate
...
 
+REGIN="us"
+
+if [ "$#" -ne 1 ]
+then
+      echo "Usage:      ./time_sync.sh <ntp_pool_regin>"
+      echo "Example:    ./time_sync.sh cn                               # Set ntp pool to China server"
+      echo "Default using us as ntp pool regin"
+else
+      REGIN=$1
```
Comment (xiaoxq) [inline scripts/time_sync.sh]: Fix indents.

File: modules/planning/on_lane_planning.cc [hunk 2]
```diff
@@ -211,7 +211,9 @@ void OnLanePlanning::RunOnce(const LocalView& local_view,
   VehicleState vehicle_state =
       VehicleStateProvider::Instance()->vehicle_state();
   const double vehicle_state_timestamp = vehicle_state.timestamp();
-  DCHECK_GE(start_timestamp, vehicle_state_timestamp);
+  DCHECK_GE(start_timestamp, vehicle_state_timestamp)
+      << "start_time_stamp is behind vehicle_state_timestamp by"
+      << start_time_stamp - vehicle_state_timestamp << "secs";
```
Comment (xiaoxq) [inline modules/planning/on_lane_planning.cc]: Leave some space before and after numbers.

File: scripts/time_sync.sh (hunk 1 above)
Comment (Capri2014) [reply scripts/time_sync.sh]: Done

File: modules/planning/on_lane_planning.cc (hunk 2 above)
Comment (Capri2014) [reply modules/planning/on_lane_planning.cc]: Done

**Pull request merged**